### PR TITLE
Aj/55/big fixes

### PR DESCRIPTION
**Pull request by webbushka**
## Description

Bug fixes for all the things in the following issue

closes: #55 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch
- ~~[ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.~~
- ~~[ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`~~
- [x] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: src/Pages/Accounts/Components/AccountAssets.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const AccountAssets = () => {
 
   // Table header values in order
   const tableHeaders = [
-    { displayName: 'Asset', dataName: 'denom' },
+    { displayName: 'Asset', dataName: 'queryDenom' },
```
Comment (crussell-figure) [inline src/Pages/Accounts/Components/AccountAssets.js:11]: Will this still show as the denom, and not the queryDenom?

File: src/Pages/Asset/Components/AssetInformation.js [hunk 2]
```diff
@@ -12,11 +12,11 @@ const AssetInformation = () => {
     getAssetInfo(assetId);
   }, [assetId, getAssetInfo]);
 
-  const { marker, ownerAddress, supply = {}, mintable, holderCount, txnCount, markerStatus } = assetInfo;
+  const { marker, holdingAccount, supply = {}, mintable, holderCount, txnCount, markerStatus } = assetInfo;
   const { circulation: supplyCirculation, total: supplyTotal } = supply;
   const summaryData = [
     { title: 'Asset Name', value: marker },
-    { title: 'Owner', value: ownerAddress, link: `/accounts/${ownerAddress}`, copy: ownerAddress },
+    { title: 'Owner', value: holdingAccount, link: `/accounts/${holdingAccount}`, copy: holdingAccount },
```
Comment (crussell-figure) [inline src/Pages/Asset/Components/AssetInformation.js:19]: Rename the title to 'Holding Account' for clarity's sake

File: src/Pages/Assets/Components/AssetsList.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ const AssetsList = () => {
     { displayName: 'Name', dataName: 'marker' },
     { displayName: 'Circulation', dataName: 'supplyCirculation' },
     { displayName: 'Supply', dataName: 'supplyTotal' },
-    { displayName: 'Owner', dataName: 'ownerAddress' },
+    { displayName: 'Owner', dataName: 'holdingAccount' },
```
Comment (crussell-figure) [inline src/Pages/Assets/Components/AssetsList.js:25]: Rename to 'Holding Account'

**Review by crussell-figure — CHANGES_REQUESTED** [review]
I also left an additional comment on the issue 
https://github.com/provenance-io/explorer-frontend/issues/55#issuecomment-828147017

File: src/Pages/Accounts/Components/AccountAssets.js (hunk 1 above)
Comment (webbushka) [reply src/Pages/Accounts/Components/AccountAssets.js:11]: Yes it's only for that link on the left
Screen Shot 2021-04-28 at 6 56 14 AM

Comment (crussell-figure) [reply src/Pages/Accounts/Components/AccountAssets.js:11]: but thats showing nhash, not hash... Thats what i mean. It should display as the denom (hash), but query as the queryDenom (nhash)

Comment (webbushka) [reply src/Pages/Accounts/Components/AccountAssets.js:11]: ok, I assumed you meant in the balance. Just pushed a change, now it shows denom in the text and uses queryDenom for the url

**Review by crussell-figure — APPROVED** [review]
LGTM!! Thank you!!

**Pull request merged**